ribbon/ribbon/encode.py

```diff
@@ -37,47 +37,47 @@
 # ---------------------------------------------------------------------------
 # Helper Functions
 # ---------------------------------------------------------------------------
-def uint_encoder(type: str) -> object:
+def uint_encoder(data_type: str) -> object:
     """
     Encoder for uint types
 
     Args:
-        type (str): Data type in string
+        data_type (str): Data type in string
 
     Returns:
         encoder (object): Uint encoder
     """
-    match = re.findall('^(u?)int(\d*)$', type).pop()
+    match = re.findall('^(u?)int(\d*)$', data_type).pop()
     signed = match[1] == ''
     width = int(match[2]) if len(match) == 3 else 256
 
     if width % 8 != 0 or width > 256 or (len(match) == 3 and match[2] != str(width)):
-        raise ValueError(f'Invalid numeric width: {type}')
+        raise ValueError(f'Invalid numeric width: {data_type}')
 
     boundsUpper = 2 ** (width - 1) - 1 if signed else 2 ** (width) - 1
-    boundsLower = (boundsUpper + 1) * (-1)
+    boundsLower = -boundsUpper - 1
```

Comment:
personally I understand better the older version :D

Reply:
Really? So the math is particularly subjective! 🙂
In this case let Ken to decide if maintain the current version or accept the change, definitely it is not an important change and I can quickly revert it 😉

Reply:
This seems fine

Reply:
cool